### PR TITLE
Documents error fix in readme file.

### PR DESCRIPTION
**Pull request by wandergithub**
# 🔗 Issue

# ✍️ Description

Adds a troubleshooting dropdown to list preview errors.

I added it under the tests section since the error listed references a test:system error encountered while running tests locally.

Hoping it helps anyone with similar issues in the future.

# 📷 Screenshots/Demos

image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,15 @@ Run ALL tests:
 ```
 ./bin/rails test:all
 ```
+## Troubleshoot
+<details>
+  <summary>Test Error</summary>
```
Comment (kasugaijin) [inline README.md:129]: Can we make it plural 'Errors'? I think we will add to it in future.

File: README.md [hunk 2]
```diff
@@ -124,6 +124,15 @@ Run ALL tests:
...
+  - System tests Error TCP Connection refused
+  
+  Fix: chromium install
+
+  [Case in point](https://github.com/rubyforgood/pet-rescue/pull/763#issuecomment-2140971709)
+</details>
```
Comment (kasugaijin) [inline README.md:135]: Can we add a space before the next heading here?

Thanks!

**Comment by wandergithub** [comment]
@kasugaijin Implemented the required changes here.

**Review by kasugaijin — APPROVED** [review]
LGTM!

**Pull request merged**